packages/react/src/AnchorNav/AnchorNav.module.css

```diff
@@ -0,0 +1,339 @@
+@media screen and (min-width: 63.25rem) {
+  .AnchorNav-link {
+    position: relative;
+    margin: 0 1px; /* to provide space for focus border */
+  }
+
+  .AnchorNav-link--center {
+    justify-content: center;
+  }
+
+  .AnchorNav-link--start {
+    justify-content: flex-start;
+    width: auto;
+    margin-right: var(--base-size-48);
+  }
+
+  .AnchorNav-link::after {
+    content: '';
+    position: absolute;
+    bottom: 10px;
+    left: 0;
+    width: 100%;
+    height: 2px;
+    background-color: var(--brand-AnchorNav-border-nav-default);
+    opacity: 0;
+    transition: opacity 300ms, transform 300ms;
+    opacity: 1;
+    transform: scale(0);
+    transform-origin: center;
+  }
+
+  .AnchorNav-link:hover::after,
+  .AnchorNav-link:focus::after,
+  .AnchorNav-link[data-active='true']::after {
+    opacity: 1;
+    transform: translate3d(0, 0.2em, 0);
+    transform: scale(0.4);
+  }
+
+  .AnchorNav-link:active::after {
+    background-color: var(--brand-AnchorNav-border-nav-pressed);
+    transform: scale(0.6);
+  }
+
+  .AnchorNav-link[data-active='true']::after {
+    background-color: var(--brand-AnchorNav-border-nav-pressed);
+    opacity: 1;
+  }
+
+  .AnchorNav-link:focus-visible::after {
+    opacity: 0;
+  }
+}
+
+/*
+ * 4. AnchorNav link labels
+ */
+.AnchorNav-link-label {
+  font-weight: var(--brand-heading-weight-400);
+  font-size: var(--brand-text-size-300);
+  line-height: var(--brand-text-lineHeight-300);
+  font-family: var(--brand-heading-fontFamily);
+  letter-spacing: var(--brand-heading-letterSpacing-400);
+  color: var(--brand-color-text-default);
+  transition: color cubic-bezier(0.33, 1, 0.68, 1) 0.2s; /* TODO: replace with global token for animation timings */
+  white-space: nowrap;
+}
+
+.AnchorNav--with-default-background-color .AnchorNav-link-label {
+  background-color: var(--brand-color-canvas-default);
+}
+
+/* Up to large breakpoint */
+@media screen and (max-width: 63.25rem) {
+  .AnchorNav-link-label {
+    background-color: var(--brand-color-canvas-default);
+  }
+
+  .AnchorNav--expanded .AnchorNav-link-label--is-active {
+    color: var(--brand-color-focus);
+  }
+
+  .AnchorNav-link:not(.AnchorNav-link[data-active='true']) {
+    visibility: hidden;
+  }
+  .AnchorNav--stuck .AnchorNav-link[data-active='false'][data-first='true'] {
+    visibility: visible;
+  }
+
+  .AnchorNav .AnchorNav-link[data-active='false'][data-first='true'] {
+    visibility: visible;
+  }
+
+  .AnchorNav-inner-container--expanded .AnchorNav-link {
+    visibility: visible !important;
+    display: flex;
+    position: relative;
+    left: auto;
+    right: auto;
+    top: auto;
+    flex-direction: column;
+    text-align: left;
+    justify-content: center;
+    align-items: flex-start;
+  }
+}
+
+/*
+ * 5. AnchorNav menu toggle button (narrow)
+ */
+.AnchorNav-menu-button {
+  display: none;
+}
+
+/* Up to large breakpoint */
+@media screen and (max-width: 63.25rem) {
+  .AnchorNav-menu-button {
+    all: unset;
```

Comment:
Depending on inheritance `all: unset` can be dangerous I believe. It could remove references to expectedly implied css. If this is a common company use then no worries.

Reply:
Glad you brought this up! No policy around this. Reason I felt it was okay here, was because I'm taking advantage of the semantics provided through `<button>` markup, but don't need any of the default styling of a button as it's not meant to look at all similar to one. I'm also manually reapplying all pseudo states, so this I don't think we're losing out on anything here, but if you feel we are, let me know.